src/EFCore.PG/Design/Internal/NpgsqlDesignTimeServices.cs

```diff
@@ -31,13 +33,19 @@
 
 namespace Npgsql.EntityFrameworkCore.PostgreSQL.Design.Internal
 {
+    /// <summary>
+    /// Enables configuring Npgsql-specific design-time services.
+    /// Tools will automatically discover implementations of this interface that are in the startup assembly.
+    /// </summary>
+    [UsedImplicitly]
     public class NpgsqlDesignTimeServices : IDesignTimeServices
     {
-        public virtual void ConfigureDesignTimeServices([NotNull] IServiceCollection serviceCollection)
+        /// <inheritdoc />
+        public virtual void ConfigureDesignTimeServices(IServiceCollection serviceCollection)
             => serviceCollection
-                .AddSingleton<IRelationalTypeMappingSource, NpgsqlTypeMappingSource>()
-                .AddSingleton<IDatabaseModelFactory, NpgsqlDatabaseModelFactory>()
-                .AddSingleton<IProviderConfigurationCodeGenerator, NpgsqlConfigurationCodeGenerator>()
-                .AddSingleton<IAnnotationCodeGenerator, NpgsqlAnnotationCodeGenerator>();
+               .AddSingleton<IRelationalTypeMappingSource, NpgsqlTypeMappingSource>()
+               .AddSingleton<IDatabaseModelFactory, NpgsqlDatabaseModelFactory>()
+               .AddSingleton<IProviderConfigurationCodeGenerator, NpgsqlConfigurationCodeGenerator>()
+               .AddSingleton<IAnnotationCodeGenerator, NpgsqlAnnotationCodeGenerator>();
```

Comment:
Indent must be a multiple of 4 everywhere.

Reply:
That's an autoformat, I'll revert.